components/items/ItemsGrid/ItemsGrid.vue

```diff
@@ -7,33 +7,74 @@
     </div>
 
     <hr />
-    <DynamicGrid>
-      <div v-for="(nft, index) in nfts" :key="`${nft.id}=${index}`">
+    <a
+      v-if="startPage > 1 && !isLoading && total > 0"
+      class="is-flex is-justify-content-center pb-4"
+      @click="reachTopHandler">
+      <b-icon icon="chevron-up" />
+    </a>
```

Comment:
just one thing, I would make this as component because we plan to reuse that across app

Reply:
DONE